common.php

```diff
@@ -32,3 +34,28 @@ function startsWith($string, $startString) {
   $len = strlen($startString);
   return (substr($string, 0, $len) === $startString);
 }
+
+function get_users_name() {
+	// Gets the first name of the logged in user
+	if(isset($_SESSION)){
+	  $fname = $_SESSION["current_user"];
+	  return $fname->first_name;
+	}else{
+	  return '';
+	}
+  }
+
+  function auth_loggedin() {
+    if(isset($_SESSION) && array_key_exists("AuthKey", $_SESSION)){ // Session is started and authentication key exists
```

Comment:
I feel like the name of the function is not a good hint of what the function does 🤔 I'm not too sure what I would name it, or it seems like the function plays the role of verifying that the user is logged in, idk maybe something like `verify_logged_in` or `verify_authentication` 😂 maybe you'll have more ideas

Reply:
Good idea, ill change it to verify_logged_in, i think that sums up what it does